package-lock.json

```diff
@@ -51,7 +51,8 @@
         "extend": {
```

Comment:
Just for my own education: this file looks like auto-generated? Does this mean that it is better to exclude this from the repository completely or does it serve another purpose?

even if we should exclude it, not suggesting in this PR, again just trying to understand/learn.

Reply:
It is auto-generated. It has changed because we've started using a new NPM version.
It should be included. More details can be found here: https://docs.npmjs.com/files/package-lock.json